portfolio/target/portfolio-1/script.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Adds a random greeting to the page.
+ */
+
+var count = 0;
+
+function addRandomGreeting() {
+  const greetings =
+      ['I ran Track and Cross Country all throughout high school',
+       'I enjoy baking',
+        'My favorite book is The Fountainhead by Ayn Rand', 
+        'I am from Guyana',
+        'My favorite food is burgers!',
+        'My current binge is Greys Anatomy'];
+
+  // Pick a random greeting.
+  if(count >= greetings.length) {
+      count = 0;
+  }
+  const greeting = greetings[count];
+  count++;
+  // Add it to the page.
+  const greetingContainer = document.getElementById('greeting-container');
+  greetingContainer.innerText = greeting;
+}
+
+// function addName() { 
```

Comment:
You can just delete the functions that you are not using right now. If you ever need it again in the future, you can always look through the file's history for it: https://github.com/bhassan6621/StepTraining/commits/63576f6b2dee5c11e6d23a3415958008c5fb0ebf/portfolio/target/portfolio-1/script.js